tests/integration/test_lambda.py

```diff
@@ -249,6 +249,91 @@ def receive_dlq():
         sqs_client.delete_queue(QueueUrl=queue_url)
         lambda_client.delete_function(FunctionName=lambda_name)
 
+    def test_failure_destination(self):
```

Comment:
nitpick: The code in these two functions seems to be almost identical - can we pull it out into a small util function, to avoid duplication? (e.g., we could create a single `test_success_and_failure_destination(..)` method, and then an inline function which we call twice with different parameters to run the two tests.)

Reply:
resolved